pkg/core/blockchain.go

```diff
@@ -1518,6 +1518,7 @@ func (bc *Blockchain) AddBlock(block *block.Block) error {
 			if err != nil && bc.config.VerifyTransactions {
 				return fmt.Errorf("transaction %s failed to verify: %w", tx.Hash().StringLE(), err)
 			}
+			bc.log.Warn(fmt.Sprintf("transaction %s failed to verify: %s", tx.Hash().StringLE(), err))
```

Comment:
`transaction xxx failed to verify: nil` for every transaction?

Reply:
Oh, my bad, got distracted.

Reply:
Fixed.